tsconfig.browser.json

```diff
@@ -1,11 +1,11 @@
 {
-  "extends": "./tsconfig.prod.json",
+  "extends": "@ethereumjs/config-typescript/tsconfig.browser.json",
   "include": ["browser/index.js"],
   "exclude": ["lib/index.js"],
   "compilerOptions": {
     "outDir": "dist.browser",
-    "target": "es5",
-    "lib": ["es5"],
-    "alwaysStrict": true
```

Comment:
These are included in the default browser config. `alwaysStrict` is a [sub-category of `strict`][1],
which TS handbook describes as:
> Enables all strict type checking options. Enabling --strict enables --noImplicitAny, --noImplicitThis, --alwaysStrict, --strictBindCallApply, --strictNullChecks, --strictFunctionTypes and --strictPropertyInitialization

[1]: https://www.typescriptlang.org/docs/handbook/compiler-options.html